lua/orgmode/org/autocompletion/omni.lua

```diff
@@ -33,7 +35,10 @@ local properties = {
 local links = {
   line_rgx = vim.regex([[\(\(^\|\s\+\)\[\[\)\@<=\(\*\|\#\|file:\)\?\(\(\w\|\/\|\.\|\\\|-\|_\|\d\)\+\)\?]]),
   rgx = vim.regex([[\(\*\|\#\|file:\)\?\(\(\w\|\/\|\.\|\\\|-\|_\|\d\)\+\)\?$]]),
-  fetcher = Hyperlinks.find_matching_links,
+  fetcher = function(url)
+    local hyperlinks, mapper = Hyperlinks.find_matching_links(url)
+    return mapper(hyperlinks)
```

Comment:
Making the mapping explicit in the omnicompletion helped me a lot to wrap my head around what actually happens here.